tests/models/imagebind/test_processor_imagebind.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import shutil
+import tempfile
+import unittest
+
+import numpy as np
+import pytest
+
+from transformers import CLIPTokenizer, CLIPTokenizerFast, ImageBindFeatureExtractor
+from transformers.testing_utils import require_torchaudio, require_vision
+from transformers.utils import is_vision_available
+
+
+if is_vision_available():
+    from PIL import Image
+
+    from transformers import ImageBindImageProcessor, ImageBindProcessor
+
+
+@require_vision
+@require_torchaudio
+class ImageBindProcessorTest(unittest.TestCase):
```

Comment:
Suggestion: add `ProcessorTesterMixin` here if possible (new feature, currently in `Align` model) for common processor testing